submissions/js/applied-js/functions/2303031087022.js

```diff
@@ -0,0 +1,59 @@
+// Task 1: Using Array Methods
+// Write a function squareNumbers(arr) using map() and arrow functions
+
+const squreNumber = (arr) => arr.map(num=>num ** 2);
+
+const numbers =[1,2,3,4,5];
+const squred = squreNumber(numbers);
+console.log(squred);
+
+
+// Task 2: Custom Filter Function
+// Create a function filterEvenNumbers(arr) using filter() and arrow functions
+
+const filterEvenNumbers = (arr) => arr.filter(num => num % 2 === 0);
+
+const numbers1 = [1, 2, 3, 4, 5, 6];
```

Comment:
if you use, let in the above declaration, no need to add const or  use a separate variable name